zk/src/archive/web/js/zk/fmt/zk.wpd

```diff
@@ -12,6 +12,7 @@ Copyright (C) 2009 Potix Corporation. All Rights Reserved.
 -->
 
 <package name="zk.fmt" language="xul/html" depends="zul.lang">
+	<script src="ext/Intl.js" browser="ie8,ie9,ie10"/>
```

Comment:
1. No need for ie8
2. I've seen a complete.js - can we use this instead of load dynamically?
https://github.com/andyearnshaw/Intl.js/blob/master/locale-data/complete.js

Reply:
yes